hack/verify-flags/excluded-flags.txt

```diff
@@ -23,3 +23,4 @@ retry_time
 file_content_in_loop
 break_on_expected_content
 Premium_LRS
+VCP_STRESS_ITERATIONS
```

Comment:
is this actually a flag?

Reply:
Its not exactly. But make-verify fails without adding this flag. i have explained the reason in comment - https://github.com/kubernetes/kubernetes/pull/57258#issuecomment-356449396

Reply:
thanks